packages/tldraw/src/hooks/useKeyboardShortcuts.tsx

```diff
@@ -15,6 +15,27 @@ export function useKeyboardShortcuts(ref: React.RefObject<HTMLDivElement>) {
     [ref]
   )
 
+  React.useEffect(() => {
+    if (!app) return
+    const handlePaste = (e: ClipboardEvent) => {
+      let items = e.clipboardData?.items ?? []
+      for (var index in items) {
+        var item = items[index]
+        if (item.kind === 'file') {
+          var file = item.getAsFile()
+          if (file) {
+            app.addMediaFromFile(file)
+          }
+        }
+      }
+    }
+
```

Comment:
Works great! copy & paste multiple images. However as @judicaelandria commented in discord size can be scaled down.


- Figma looks like doesn't scale down the image (but tldraw default scaling looks bigger)
- When pasting multiple images, tldraw placing one image on top of other. However Figma places side by side (better UX)


<img width="197" alt="image" src="https://user-images.githubusercontent.com/354596/167030315-ef9f4164-b679-4f8f-bf41-3c13e0d270cb.png">

Reply:
scaling down the image but preserving the aspect ratio

Reply:
This would be a good separate issue!